### PR TITLE
feat(devtools): support performance tab for tracing protocol

### PR DESCRIPTION
**Pull request by lavnFan**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

File: devtools/devtools-backend/include/module/domain_propos.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ constexpr char kFrontendKeyDomainNameTDFInspector[] = "TDFInspector";
 constexpr char kFrontendKeyDomainNameTDFMemory[] = "TDFMemory";
 constexpr char kFrontendKeyDomainNameTDFPerformance[] = "TDFPerformance";
 constexpr char kFrontendKeyDomainNameTDFRuntime[] = "TDFRuntime";
+constexpr char kFrontendKeyDomainNameTracing[] = "Tracing";
```
Comment (churchill-zhang) [inline devtools/devtools-backend/include/module/domain_propos.h:34]: 为啥前面几个是TDFXXX，这个又没有TDF呢

File: driver/js/core/src/devtools/trace_control.cc [hunk 2]
```diff
@@ -81,7 +83,9 @@ std::string TraceControl::GetTracingContent() {
     if (!tracing_content.empty() && tracing_content[0] == ',') {
       tracing_content = tracing_content.substr(1);
     }
-    return kTraceBeginTag + tracing_content + kTraceEndTag;
+    std::string result = "{\"";
```
Comment (churchill-zhang) [inline driver/js/core/src/devtools/trace_control.cc:86]: 从不授信的文件中读内容需不需要有安全检查？

Comment (churchill-zhang) [reply driver/js/core/src/devtools/trace_control.cc:86]: 这里是给V8用的吗？如果v8用可以让v8内部负责

**Review by churchill-zhang — APPROVED** [review]

File: devtools/devtools-backend/include/module/domain_propos.h (hunk 1 above)
Comment (lavnFan) [reply devtools/devtools-backend/include/module/domain_propos.h:34]: 这个非自定义协议，是 CDP 标准协议

File: driver/js/core/src/devtools/trace_control.cc (hunk 2 above)
Comment (lavnFan) [reply driver/js/core/src/devtools/trace_control.cc:86]: 这个是从v8拿到的数据给frotnend 用

**Review by siguangli — APPROVED** [review]

**Review by ilikethese — APPROVED** [review]

**Review by medns — APPROVED** [review]
LGTM

**Pull request merged**